rules/creates-box-access-token.md

```diff
@@ -40,15 +42,17 @@ function(user, context, callback) {
   // 3. Save the ID in the box_id property for next time
   // 4. Get a Box User access_token
   
-  getBoxAccessToken(boxClientId, boxClientSecret, 
-                    boxEnterpriseId, "enterprise", privateKey, function(e,token){
+  getBoxAccessToken(boxClientId, boxClientSecret, boxEnterpriseId, "enterprise", privateKey, function(e,token){
     createUser(user, token.access_token, function(e,u){
-      if(e) return callback(e,user,context);
-      user.persistent.box_id = u.id; //Save property in users' metadata
-      getBoxAccessToken(boxClientId, boxClientSecret, 
-                        u.id, "user", privateKey, function(e,token){
-        if(e) return callback(e,user,context);
-        user.box_access_token = token;
+      if(e) {
+        return callback(e,user,context);
+      }
+      user.app_metadata.box_id = u.id; //Save property in users' metadata
```

Comment:
I believe so yes. There is a check on Line 28 to see whether user has already has been provisioned in Box. It looks at `user.app_metadata.box_id` to see whether this has been done